tests/comboreduct/converter/combo_atomeseUTest.cxxtest

```diff
@@ -0,0 +1,68 @@
+/** combo_atomeseUTest.cxxtest ---
+ *
+ * Copyright (C) 2018 OpenCog Foundation
+ *
+ * Author: Kasim Ebrahim
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License v3 as
+ * published by the Free Software Foundation and including the exceptions
+ * at http://opencog.org/wiki/Licenses
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program; if not, write to:
+ * Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+
+#include <moses/comboreduct/converter/combo_atomese.h>
+#include <moses/comboreduct/combo/combo.h>
+#include <opencog/atoms/base/Handle.h>
+#include <opencog/atoms/base/atom_types.h>
+#include <opencog/atoms/base/Link.h>
+#include <opencog/atoms/base/Node.h>
+
+
+using namespace std;
+using namespace opencog;
+
+
+class combo_atomeseUTest : public CxxTest::TestSuite{
+
+public:
+    void test_atomese_combo_predicate(void){
+        combo::combo_tree tr = combo::str2combo_tree("and(or($1 $2) not($3))", {});
+
+        Handle h = combo::atomese_combo(tr);
+
+        HandleSeq list_or = {createNode(PREDICATE_NODE, "1"), createNode(PREDICATE_NODE, "2")};
+        Handle or_link = createLink(list_or, OR_LINK);
```

Comment:
The code is fine, just note that there are other Link constructors that you might find useful, that may for instance create more compact code, see https://github.com/opencog/atomspace/blob/master/opencog/atoms/base/Link.h#L77